pkg/cloud/amazon/s3_storage_test.go

```diff
@@ -64,13 +64,9 @@ func skipIfNoDefaultConfig(t *testing.T, ctx context.Context) {
 	t.Helper()
 	const helpMsg = "we only run this test if a default role exists, " +
 		"refer to https://docs.aws.com/cli/latest/userguide/cli-configure-role.html"
-	config, err := config.LoadDefaultConfig(ctx,
-		config.WithSharedConfigProfile(config.DefaultSharedConfigProfile))
-	if err != nil && err.Error() == "failed to get shared config profile, default" {
-		skip.IgnoreLintf(t, "%s: %s", helpMsg, err)
-	}
```

Comment:
Same reasoning here — we should not enforce that a `DefaultSharedConfigProfile` exists as for auth implicit tests it shouldn't need to